app/api/sentry-test/route.ts

```
@@ -0,0 +1,55 @@
+        // Test async error
+        await new Promise((resolve, reject) => {
+          setTimeout(() => {
+            reject(new Error("Test Sentry Async Error - This async error should be caught by Sentry"));
+          }, 100);
+        });
+        break;
+        
+      case "reference":
+        // Test reference error
+        // @ts-ignore - Intentional error for testing
+        const obj = null;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)